rbacutils/rbacprocessor.go

```diff
@@ -82,7 +84,7 @@ func InitSAID2WLIDmap(k8sAPI *k8sinterface.KubernetesApi, clusterName string) (m
 	return saID2WLIDmap, nil
 }
 
-//TODO - DEPRECATE sa2WLIDmap
+// TODO - DEPRECATE sa2WLIDmap
```

Comment:
Could this be removed?

Reply:
yes, we'll need to deprecate it in ks and BE as well